specification/iothub/resource-manager/Microsoft.Devices/stable/2018-04-01/examples/iothub_testallroutes.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "iotHubName": "testHub",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2018-04-01",
+    "subscriptionId": "91d12660-3dec-467a-be2a-213b5544ddc0",
+    "input": {
+      "routingSource": "DeviceMessages",
+      "message": {
+        "body": "Body of message",
+        "appProperties": "App Properties",
+        "systemProperties": "System Properties"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "routes": [
+          {
+            "properties": {
+                "name": "Route id",
+                "source": "DeviceMessages",
+                "endpointNames": ["id1", "id2"],

```

Comment:
Sorry for the late response. Here is another issue I found - this array is defined to have max length of 1: https://github.com/Azure/azure-rest-api-specs/pull/3031/files#diff-54dafbe4198f749bac837b913372bcddR2417